enzy_htp/analysis/electric_field.py

```diff
@@ -0,0 +1,82 @@
+"""Submodule contains code for calculating the electric field strength in an enzyme system along a specified
```

Comment:
The current function seems only designed to be working on a single structure. In real cases, EF are also calculated from trajectories. I will work on design classes to contain trajectory/ensemble (maybe a class hold structures: List[Structure]?) and maybe we can add a dispatch to this function in the future.

Reply:
No changes needed here